Content.Shared/Throwing/BeforeThrowEvent.cs

```diff
@@ -1,20 +1,22 @@
 using System.Numerics;
 
-namespace Content.Shared.Throwing
+namespace Content.Shared.Throwing;
+
+[ByRefEvent]
+public struct BeforeThrowEvent
 {
-    public sealed class BeforeThrowEvent : HandledEntityEventArgs
+    public BeforeThrowEvent(EntityUid itemUid, Vector2 direction, float throwStrength,  EntityUid playerUid)
```

Comment:
Thank you, I missed that.

Reply:
I’ve documented this in the PR description, please let me know if that isn’t enough detail.

Reply:
Thanks. Ideally you also show how to fix this, e.g. "if you handle this event please add `ref`" in front of the handler, but this is good enough. Otherwise this looks fine and I intend to merge as soon as I get around to giving it a quick test. Thanks for the contribution :)

Reply:
Fantastic, it’s been delightful working with everyone. :D